onnxruntime/core/providers/cpu/ml/tree_ensemble_common.h

```diff
@@ -957,6 +963,396 @@ Status TreeEnsembleCommonClassifier<InputType, ThresholdType, OutputType>::compu
+        target_class_treeids_old, target_class_weights_as_tensor_old);
+  }
+  return false_nodeid;
+}
+
+template <typename IOType, typename ThresholdType>
+void TreeEnsembleCommonV5<IOType, ThresholdType>::transformInputAllTrees(
+    // input
+    const std::vector<int64_t>& leaf_targetids, const std::vector<ThresholdType>& leaf_weights, const std::vector<std::vector<ThresholdType>>& membership_values_by_id,
+    const std::vector<int64_t>& nodes_falseleafs, const std::vector<int64_t>& nodes_falsenodeids, const std::vector<int64_t>& nodes_featureids,
+    const std::vector<ThresholdType>& nodes_hitrates, const std::vector<int64_t>& nodes_missing_value_tracks_true, const std::vector<uint8_t>& nodes_modes,
+    const std::vector<ThresholdType>& nodes_splits, const std::vector<int64_t>& nodes_trueleafs, const std::vector<int64_t>& nodes_truenodeids,
+    const std::vector<int64_t>& tree_roots,
+    // output
+    std::vector<int64_t>& nodes_falsenodeids_old, std::vector<int64_t>& nodes_featureids_old, std::vector<ThresholdType>& nodes_hitrates_as_tensor_old,
+    std::vector<int64_t>& nodes_missing_value_tracks_true_old, std::vector<uint8_t>& nodes_modes_old, std::vector<int64_t>& nodes_nodeids_old,
+    std::vector<int64_t>& nodes_treeids_old, std::vector<int64_t>& nodes_truenodeids_old, std::vector<ThresholdType>& nodes_values_as_tensor_old,
+    std::vector<int64_t>& target_class_ids_old, std::vector<int64_t>& target_class_nodeids_old, std::vector<int64_t>& target_class_treeids_old,
+    std::vector<ThresholdType>& target_class_weights_as_tensor_old) {
+  int64_t curr_treeid = 0;
+  for (const int64_t& tree_root : tree_roots) {
+    size_t tree_root_size_t = onnxruntime::narrow<size_t>(tree_root);
+    transformInputOneTree(tree_root_size_t, curr_treeid, 0, 0U, nodes_falsenodeids[tree_root_size_t] == nodes_truenodeids[tree_root_size_t],
+                          leaf_targetids, leaf_weights, membership_values_by_id, nodes_falseleafs, nodes_falsenodeids, nodes_featureids,
+                          nodes_hitrates, nodes_missing_value_tracks_true, nodes_modes, nodes_splits, nodes_trueleafs, nodes_truenodeids, tree_roots,
+                          nodes_falsenodeids_old, nodes_featureids_old, nodes_hitrates_as_tensor_old, nodes_missing_value_tracks_true_old, nodes_modes_old,
+                          nodes_nodeids_old, nodes_treeids_old, nodes_truenodeids_old, nodes_values_as_tensor_old, target_class_ids_old, target_class_nodeids_old,
+                          target_class_treeids_old, target_class_weights_as_tensor_old);
+    curr_treeid++;
+  }
+}
+
+template <typename IOType, typename ThresholdType>
+Status TreeEnsembleCommonV5<IOType, ThresholdType>::Init(const OpKernelInfo& info) {
+  std::vector<ThresholdType> leaf_weights, nodes_hitrates, membership_values, nodes_splits;
+  std::vector<uint8_t> nodes_modes;
+
+  ORT_THROW_IF_ERROR(GetVectorAttrsOrDefault(info, "leaf_weights", leaf_weights));
```

Comment:
You still have an issue here. GetVectorAttrsOrDefault was excluded from the minimal build (#if !defined(ORT_MINIMAL_BUILD)). You may raise an exception in that case saying TreeEnsemble v5 is not available. This flag is not enabled for the python package.